views/js/GoogleAnalyticActionLib.js

```diff
@@ -76,22 +76,19 @@ var GoogleAnalyticEnhancedECommerce = {
 	addToCart: function(Product) {
 		this.add(Product);
 		ga('ec:setAction', 'add');
-		ga('send', 'event', 'UX', 'click', 'Add to Cart'); // Send data using an event.
+		ga('send', 'event', 'UX', 'click', 'Add to Cart');
 	},
 
 	removeFromCart: function(Product) {
 		this.add(Product);
 		ga('ec:setAction', 'remove');
-		ga('send', 'event', 'UX', 'click', 'Remove From cart'); // Send data using an event.
+		ga('send', 'event', 'UX', 'click', 'Remove From cart');
 	},
 
 	addProductImpression: function(Product) {
-		//ga('send', 'pageview');
 	},
```

Comment:
Maybe you can delete this function instead of keeping it empty?

Reply:
Isn't it a BC break?

Reply:
Nope since we don't define HTML, CSS and JS as part of our public API. Otherwise, everything is a BC breaks.